sdk/communication/azure-communication-email/azure/communication/email/_shared/utils.py

```diff
@@ -9,12 +9,10 @@
 import calendar
 from typing import (cast,
                     Tuple,
-                    Union,
                     )
 from datetime import datetime
 from msrest.serialization import TZ_UTC
-from azure.core.credentials import AccessToken
-
+from azure.core.credentials import AccessToken, AzureKeyCredential
```

Comment:
The changes to the _shared directory will need to be synced across the other communications SDKs